### PR TITLE
Improvement/wallet connect 2.x

### PR DESCRIPTION
**Pull request by IAmKio**
## Description

-

## Motivation and Context

-

## How Has This Been Tested?

-
-
-

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: src/services/walletConnect.js [hunk 1]
```diff
@@ -39,9 +56,42 @@ const clientMeta = {
 };
 /* eslint-enable i18next/no-literal-string */
 
-export const createConnector = (
-  options: WalletConnectOptions,
-): ?WalletConnectConnector => {
+export async function createWeb3Wallet() {
+  core = new Core({
+    projectId: getEnv().PROJECT_ID,
```
Comment (IAmKio) [inline src/services/walletConnect.js:61]: Remember to change the `PROJECT_ID` name

File: src/services/walletConnect.js [hunk 2]
```diff
@@ -39,9 +56,42 @@ const clientMeta = {
...
+    core,
+    metadata: clientMeta,
+  });
+
+  return web3wallet;
+}
+
+export async function web3WalletInit() {
+  const newCore = new Core({
+    projectId: getEnv().PROJECT_ID,
```
Comment (IAmKio) [inline src/services/walletConnect.js:74]: As above

File: src/services/walletConnect.js [hunk 3]
```diff
@@ -39,9 +56,42 @@ const clientMeta = {
...
+  });
+
+  const wcInit = await Web3Wallet.init({
```
Comment (IAmKio) [inline src/services/walletConnect.js:77]: We have two `Web3Wallet.init` calls here - can we not reuse this in some way? I'm just worried that we end up with two instances of Web3Wallet which may cause connection issues

Comment (IAmKio) [reply src/services/walletConnect.js:77]: Please let me know if there was a reason for the above

Comment (lalitkumarrock) [reply src/services/walletConnect.js:77]: The session disconnect method in need new instance. So I made it separate. first one (createWeb3Wallet) used for every methods.

Comment (IAmKio) [reply src/services/walletConnect.js:77]: Understood, thanks!

File: src/configs/buildConfig.js [hunk 4]
```diff
@@ -38,7 +38,7 @@ export const buildEnvironment = {
   STORYLY_TOKEN: '_storyly_token_',
   KOCHAVA_IOS_ID: '_kochava_ios_id_',
   KOCHAVA_ANDROID_ID: '_kochava_android_id_',
-  PROJECT_ID: '15fcfb7323fcce5aa1b58afe4dc6d847',
+  WALLETCONNECT_PROJECT_ID: '15fcfb7323fcce5aa1b58afe4dc6d847',
```
Comment (IAmKio) [inline src/configs/buildConfig.js:41]: Sorry just remember to change `15fcfb7323fcce5aa1b58afe4dc6d847` to `_wallet_connect_id_` here

Comment (lalitkumarrock) [reply src/configs/buildConfig.js]: Updated it!

**Review by IAmKio — COMMENTED** [review]
LGTM

**Review by lalitkumarrock — APPROVED** [review]
LGTM!

**Pull request merged**